### ISSUE TITLE
"overrides" section from included ruleset is ignored

### ISSUE THREAD

**Issue by anikitin** [stoplightio/spectral#2331]
**Describe the bug**
I have a master ruleset file which includes other files via "extends".
I need to define some "overrides" rules and I want to place them in a separate file and add it via "extens".
So, when I define the "overrides" section in external file, it is not applied.
If I do the same in master file, it works as expected.

**To Reproduce**

1. Create two ruleset files where ruleset-1 includes ruleset-2 via "extends"
2. Define "overrides" section in ruleset-2

**Actual behavior**
Overrides from included file are not applied

**Expected behavior**
Overrides from included file should be applied

**Environment:**
 - spectral-cli 6.6.0

**Comment by P0lip** [comment]
Hey!
That is expected. Overrides aren't inherited.

**Comment by anikitin** [comment]
Hm, is there a reason behind it? Is it stated in the documentation somewhere?
I haven't found anything about this in https://docs.stoplight.io/docs/spectral/293426e270fac-overrides and https://docs.stoplight.io/docs/spectral/83527ef2dd8c0-extending-rulesets. 

What other sections will NOT be picked up from the ruleset files connected through "extends"?

**Comment by anikitin** [comment]
It seems that #2237 is about similar inheritance problem with aliases...

**Comment by philsturgeon** [comment]
A good outcome here would be to either let overrides be inhereted, or document why overrides are not being inhereted. Making it clear either way is the main thing. @mnaumanali94 @heitortsergent

**Comment by P0lip** [comment]
If you really do want to overrides to be be inherited, you can use JS rulesets, and so something as follows

```ts
import rulesetA from './ruleset';

export default {
  extends: rulesetA,
  overrides: rulesetA.overrides,
};
```

The reason they are not inherited is mostly to avoid confusion. Overrides are usually specific to a particular project you lint, so having a shareable ruleset with some overrides in it doesn't make much sense.

**Comment by philsturgeon** [comment]
Perfect, if we could put that into documentation then there won’t be any confusion, and we’re offering people a workaround if they really want it.

**Comment by heitortsergent** [comment]
I'll update the docs with the workaround and the reason. :)
